examples/rmrk/resource/wasm/tests/resources.rs

```diff
@@ -373,4 +464,55 @@ impl<'a> Fixture<'a> {
         .concat();
         program.send_bytes(actor_id, encoded_request)
     }
+
+    async fn add_resource_using_client(
+        &'a self,
+        actor_id: ActorId,
```

Comment:
Let's keep it `u64` in order to avoid ugly code in the test itself

Reply:
renamed to `program_id: ActorId`